modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc

```diff
@@ -14,6 +14,11 @@ After you have completed the preceding procedure, follow these steps to configur
 
 * Host the `rootfs` and `iso` images on an HTTPD server.
 
+[WARNING]
+====
+If you enable TLS for the HTTPD server, verify the trusted certificate chain between your {product-title} hub and spokes and the HTTPD server. Failure to implement a trusted certificate chain can result in the image creation failing to download. Using untrusted HTTPS servers is not supported.
```

Comment:
> the HTTPD server

I think this would either be written "the HTTP server" or "httpd". HTTP is the protocol that the server implements, httpd (HTTP daemon) is the process that runs on the server.

In either case I think "httpd" is typically written in lower case as far as I can tell from the [httpd documentation](https://httpd.apache.org/docs/trunk/getting-started.html). Also `httpd` usually refers specifically to Apache httpd, which isn't technically required in this case (there are other HTTP server implementations).

> verify the trusted certificate chain between your {product-title} hub and spokes and the HTTPD server

This is not particularly helpful. Someone could ensure that the certificate chain is valid but if the root certificate isn't signed by an authority that our client trusts out of the box we will still fail to download the images.

> Failure to implement a trusted certificate chain can result in the image creation failing to download.

This doesn't make sense. Maybe "Using a server configured with an untrusted certificate will prevent the images from being downloaded to the image creation service"

Reply:
@carbonin thanks so much for the feedback! To answer your questions, the information you've commented on came from the Jira ticket for [TELCODOCS-450](https://issues.redhat.com/browse/TELCODOCS-450):
> ⚠️ Warning: If you  enable TLS for the HTTPD server, ensure your OCP Hub and spokes contain the proper CA to trust the server, if not the Image creation will fail on download. Using untrusted HTTPS servers is not supported.

Reply:
Updated to "HTTPD server" for consistency with the rest of the module.